### ISSUE TITLE
menu: crash on block without content (?)

### ISSUE THREAD

**Issue by qfox** [bem/bem-components#1320]
I have bemjson

``` js
{ block : 'menu', content : { block : 'not-menu-item' } }
```

It doesn't even be rendered and fails with some dirty error in compiled file ;-(

```
TypeError: Cannot read property '0' of undefined
    at __$b29.iterateItems__$69 (.../index.bemhtml.js:501:46)
    ...
```

Can we do something with this while we can't build source maps and resolve real filename and real string?

**Comment by tadatuta** [comment]
@zxqfox I know that's a pain but you may build templates in dev mode and find the very line

**Comment by qfox** [comment]
@tadatuta I think we should notice in docs that `menu` supports ONLY objects with `content` field: https://github.com/bem/bem-components/blob/v2/common.blocks/menu/menu.bemhtml#L18 Or just fix that ;-)

**Comment by hcodes** [comment]
/cc @innabelaya 
Лучше добавить в документацию.
https://github.com/bem/bem-components/blob/v2/common.blocks/menu/menu.en.md#custom-fields-of-the-block
`content` `Array` The array of the menu items.

https://github.com/bem/bem-components/blob/v2/common.blocks/menu/menu.ru.md#custom-fields-of-the-block
`content` `Array` Массив пунктов меню.

**Comment by qfox** [comment]
@hcodes у нас часто видно assertы ругаются. В случае меню с элементами — может тоже добавим?

**Comment by hcodes** [comment]
Хорошо, добавлю.

**Comment by hcodes** [comment]
Fixes #1335

**Comment by innabelaya** [comment]
@hcodes добавила описание #1336
